seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
+    """ForwardModel Class:
+
+    Abstract base class for the SEIR model class.
+
+    Parameters
+    ----------
+    value: numeric, optional
+        example of value
+
+    """
+
+    def __init__(self):
+
+    def simulate(self):
+        """
+        Forward simulate the daily number of infections for a given time period with given parameters
```

Comment:
Could you make this a little less specific and just say that this returns the forward simulation of a model for a given time period with given parameters? It may be that we will have a model that returns other things than the number of infections, and then the doc string wouldn't be appropriate anymore.